### PR TITLE
Deploy RouterBalancerCurve

### PR DESCRIPTION
**Pull request by Preston-Harrison**

### CODE REVIEW

File: scripts/deploy/contracts/RouterBalancerCurve.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { deployContract, getChainIdForNetwork } from "../utils";
+import { ethers } from "ethers";
+import { RouterBalancerCurve } from "../../../build/typechain/RouterBalancerCurve";
+
+const constructorArguments = [process.env.ARBITRUM_BALANCER_VAULT];
```
Comment (brettvniekerk) [inline scripts/deploy/contracts/RouterBalancerCurve.ts:5]: throw error if `undefined`?

**Review by brettvniekerk — CHANGES_REQUESTED** [review]
One thing

File: scripts/deploy/contracts/RouterBalancerCurve.ts (hunk 1 above)
Comment (Preston-Harrison) [reply scripts/deploy/contracts/RouterBalancerCurve.ts:5]: will throw because it is not a valid address if undefined

Comment (brettvniekerk) [reply scripts/deploy/contracts/RouterBalancerCurve.ts:5]: ah so it throws loudly anyway, 👍

**Review by nabsta-eth — APPROVED** [review]

**Review by rellfy — APPROVED** [review]

**Review by brettvniekerk — APPROVED** [review]

**Pull request merged**